io/net/irqbalance.py

```diff
@@ -170,8 +170,7 @@ def get_ping_process_pid(self):
                                             ignore_status=True,
                                             sudo=True).decode("utf-8")
         if not process_pid:
-            self.log.debug(f"No more process PID avaialable")
-            return False
+            self.cancel(f"No process PID of ping command avaialable")
```

Comment:
@FarooqAbdulla02 if no process are running means, is this something failure?

Reply:
@Naresh-ibm No, its not failure but its unable to find the PID hence it cancels the test case.
basically we are trying to get a  PID by running ping test with " -f " option to remote machine.
by using above generated PID we will be assigning dif CPU's by using "taskset" command.

when we gave the minimal count of ping packets  the process get exited and return "false" since there is no PID available.
 
The default ping count in YAML is "10000" but when user defined something like 100, will encounter this error.

Reply:
ok